.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: java
+dist: bionic
+jdk: openjdk11
+env:
+  global:
+    - PYENV_VERSION=3.7
+install:
+  - pip3 install --user pre-commit
+script:
+  - pre-commit run --all-files --verbose
+  - mvn clean verify
```

Comment:
@rjlohan what's the difference between this and the previous `mvn compile test`?

Reply:
Practically, nothing in this repo. But `verify` would also check code coverage requirements and such if they were in the `pom.xml`

Reply:
Build lifecycle reference: https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html

Reply:
@rjlohan thoughts on adding something like this to test it working end-to-end?
```bash
mvn clean package
echo 'settings:
  regions: [us-east-1]
  output: cfn-schemas' > cfg.yml
java -jar target/aws-cloudformation-template-schema-1.0-SNAPSHOT-jar-with-dependencies.jar --config-file cfg.yml
```